Source/Scene/PointPrimitiveCollection.js

```diff
@@ -892,21 +909,22 @@ define([
             vaLength = va.length;
 
             colorList.length = vaLength;
-            for (j = 0; j < vaLength; ++j) {
+            for (j = 0; j < vaLength * 2; ++j) {
                 command = colorList[j];
                 if (!defined(command)) {
-                    command = colorList[j] = new DrawCommand({
-                        primitiveType : PrimitiveType.POINTS,
-                        pass : Pass.OPAQUE,
-                        owner : this
-                    });
+                    command = colorList[j] = new DrawCommand();
                 }
 
+                command.primitiveType = PrimitiveType.POINTS;
+                command.pass = (j % 2 === 0) ? Pass.OPAQUE : Pass.TRANSLUCENT;
```

Comment:
Let's make a public enum with the following properties:
* `OPAQUE`
* `TRANSLUCENT`
* `OPAQUE_AND_TRANSLUCENT`

Billboard, label, and point collections can expose this property (get/set ideally) and default to `OPAQUE_AND_TRANSLUCENT`.  Make sure the reference doc clearly defines the performance vs. visual quality tradeoffs.